master/buildbot/config.py

```diff
@@ -201,8 +202,9 @@ def loadConfig(cls, basedir, filename):
                 raise errors
             except Exception:
                 log.err(failure.Failure(), 'error while parsing config file:')
-                error("error while parsing config file: %s (traceback in logfile)" %
-                      (sys.exc_info()[1],),
+                error("error while parsing config file "
+                      "(full traceback in logfile):\n%s" %
```

Comment:
I agree parse error should be in the stderr.

but it looks like now the full traceback is not only in the logfile

so I would remove (full traceback in logfile)